python/CMakeLists.txt

```diff
@@ -28,7 +28,7 @@ add_custom_command(
   COMMAND ${PYTHON_EXECUTABLE}
   ARGS "setup.py" "build_ext" "--inplace"
   WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
-  DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/setup.py ${CMAKE_CURRENT_BINARY_DIR}/xcfun_swigPYTHON_wrap.c
+  DEPENDS xcfun-static ${CMAKE_CURRENT_BINARY_DIR}/setup.py ${CMAKE_CURRENT_BINARY_DIR}/xcfun_swigPYTHON_wrap.c
   )
 
```

Comment:
To avoid the install step you will need to copy `xcfun.py` to `${CMAKE_CURRENT_BINARY_DIR}`:
```
file(COPY xcfun.py DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
```
should do it. The install path will probably have to be reworked at a later stage.